.circleci/config.yml

```diff
@@ -82,6 +82,7 @@ jobs:
           command: |
             export SUDO=sudo
             $SUDO apt update
```

Comment:
@algojack Asking so that I know what to consider in-scope:  Is a `CHANGELOG.md` update expected in the PR?

Reply:
good catch, our instructions needed a bit of updating.